board/views.py

```diff
@@ -0,0 +1,29 @@
+from django.shortcuts import render
+from django.views.generic import ListView, CreateView, UpdateView
+from django.urls import reverse_lazy
+
+from .models import Person, City
+from .forms import PersonForm
+
+
+class PersonListView(ListView):
+    model = Person
+    context_object_name = 'people'
+
+
+class PersonCreateView(CreateView):
+    model = Person
+    form_class = PersonForm
+    success_url = reverse_lazy('person_changelist')
+
+
+class PersonUpdateView(UpdateView):
+    model = Person
+    form_class = PersonForm
+    success_url = reverse_lazy('person_changelist')
+
+
+def load_cities(request):
+    country_id = request.GET.get('country')
+    cities = City.objects.filter(country_id=country_id).order_by('name')
+    return render(request, 'board/city_dropdown_list_options.html', {'cities': cities})
```

Comment:
loading the cities by first a GET request of the country from the form and then filtering by that `country_id` and rendering the `city_dropdown_list_options.html` with the appropriate cities